xAPI.md

```diff
@@ -1226,8 +1226,9 @@ authors and/or organizations.
 * Upon receiving a Statement with an Activity Definition that differs from the one stored, an LRS
 SHOULD decide whether it considers the Activity Provider to have the authority to change the definition and
 SHOULD update the stored Activity Definition accordingly if that decision is positive.
-* An LRS MAY accept small corrections to the Activity’s definition. For example, it would be okay for an LRS
-to accept spelling fixes, but not to accept changes to correct responses.
+* An LRS MAY make small corrections to the Activity’s definition when recieving a new definition e.g. to fix a spelling.
```

Comment:
Another change that should probably be included: instead of "the Activity's definition", "the LRS's canonical definition for the Activity" for consistency with usage of canonical and that there isn't a single concept of "the Activity's definition".